sktime/regression/kernel_based/_rocket_regressor.py

```diff
@@ -220,4 +219,6 @@ def get_test_params(cls, parameter_set="default"):
             instance.
             ``create_test_instance`` uses the first (or only) dictionary in ``params``.
         """
-        return {"num_kernels": 20}
+        return [{"num_kernels": 20}, {"num_kernels": 100}]
+
+
```

Comment:
Usually, it's recommended to have one empty line at end. It seems now there are 3: one was already there plus you are adding two more.

I'm not sure if your local pre-commit setup is correct. Can you please follow the setup and run the following command against your branch?

```
pre-commit run --all-files
```

It's very likely going to modify the file you changed. If it modifies other files too, please ignore those changes and undo before staging and committing this specific file.